development/docker-compose.yml

```diff
@@ -1,14 +1,25 @@
-web:
-  image: metrilo.azurecr.io/magento1:latest
-  ports:
-    - "80:80"
-  links:
-    - db
-  env_file:
-    - env
-  volumes:
-    - ../:/plugin
-db:
-  image: mysql:5.6.23
-  env_file:
-    - env
+version: '3.0'
+services:
+  web:
+    image: metrilo.azurecr.io/magento1:latest
+    container_name: magento1
+    ports:
+      - "80:80"
+    links:
+      - db
+    env_file:
+      - env
+    volumes:
+      - magento1:/var/www
+      - ../:/plugin
+  db:
+    image: mysql:5.6.23
```

Comment:
Why not `5.6`? The `5.6` tag in docker hub is maintained regularly and is pointing to a later version `5.6.39`.
`5.6.23` seems unavailable in the docker repository.

Reply:
👍